main.py

```diff
@@ -39,9 +39,9 @@ def main(self): # The main function contains the most important code
         self.slowType("Made by: Drillenissen#4268 && Benz#4947", .02) # Print who developed the code
         time.sleep(1) # Wait a little more
         self.slowType("\nInput How Many Codes to Generate and Check: ", .02, newLine = False) # Print the first question
-
+        global num # added to track the amount of codes checked
```

Comment:
I would never personally use the global keyword, if you could use OOP insted that would be nice

Reply:
Yh but id dint want to rewrite it all